docs/en/reference/events.rst

```diff
@@ -825,7 +825,19 @@ An entity listener is a lifecycle listener class used for an entity.
 
 .. configuration-block::
 
-    .. code-block:: php
+    .. code-block:: attribute
+    
+        <?php
+        namespace MyProject\Entity;
+        use App\EventListener\UserListener;
+
+        #[Entity]
+        #[EntityListeners([UserListener::class])]
+        class User
+        {
+            // ....
+        }
+    .. code-block:: annotation
```

Comment:
This defines the code coloration and therefore will not work.

Reply:
Don't know what you mean :-( What I'm trying to create is basically the same as in https://github.com/doctrine/doctrine-website/pull/409

Reply:
mb it looks supported!